project-base/app/src/Model/Customer/User/RegistrationDataFactory.php

```diff
@@ -61,4 +64,25 @@ public function createWithArgument(Argument $argument): RegistrationData
 
         return $registrationData;
     }
+
```

Comment:
It seems like you should be able to move this file into the Framework package easily. If you do that it will be easier to maintain this file for us.